frontend/pages/fullscreen/playback/index.vue

```diff
@@ -60,13 +59,10 @@ export default Vue.extend({
   computed: {
     ...mapStores(pageStore, playbackManagerStore),
     backdropHash: {
-      get(): string {
-        return (
-          this.getBlurhash(
-            this.playbackManager.getCurrentItem,
-            ImageType.Primary
-          ) || ''
-        );
+      get(): string | undefined {
```

Comment:
I know you liked that style of computing methods, but what about now? Don't you think we could drop the "get" when there's no setter?

Reply:
@ThibaultNocchi I agree, however not sure how necessary the effort to change them all is if we're near the Vue 3 switch.
Most (if not all) the computed properties would need a rewrite anyway with Composition API.

Reply:
Oh no ofc not all of them at one, but in a general manner when seeing one of them. Like this one ^^